### PR TITLE
Features/reseeding db

### PR DESCRIPTION
**Pull request by cbell98**
## Description
https://www.loom.com/share/f53a65f0fc034414bf7ecae2154cf723 

Updated the Mentor and Mentee classes in the user_generators.py file to reflect the current schema.  Added 5 new data options to the data_options.py file to help generate new fields.  Preparing the MongoDB database to be reseeded. 

## Type of change

- [x] New feature

## Checklist:

- [x] My code follows PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: data_generators/data_options.py [hunk 1]
```diff
@@ -324,6 +324,35 @@
     "GCA Help", "Resume Help", "Job Search", "Progress Check"
 )
 
+states = (
```
Comment (mikayla-kosmala) [inline data_generators/data_options.py:327]: data_option.py file looks good!

**Review by mikayla-kosmala — CHANGES_REQUESTED** [review]
Can you change the tech_stack, skill_level options to reflect what is currently on the Intake forms.
Also remove preferred_mentee_exp_level from the Mentor class. Other than that it looks pretty good!

**Review by GermanParra — APPROVED** [review]

**Review by mikayla-kosmala — APPROVED** [review]
Approved! Good Job!

**Review by jinjahninjah — APPROVED** [review]
:shipit:

**Review by GermanParra — CHANGES_REQUESTED** [review]
In the schema.py we should probably delete the following fields also.
- Mentor -> 'preferred_mentee_exp_level'
- MentorUpdate -> 'preferred_mentee_exp_level'  and 'profile_id'
- Mentee -> ' experience_level'
- MenteeUpdate ->  ' experience_level' and 'profile_id'

**Comment by mikayla-kosmala** [comment]
I believe we still need mentee experience level. Not sure exactly what we will be doing with it, but we need it for now.

**Comment by GermanParra** [comment]
> In the schema.py we should probably delete the following fields also.
> 
> * Mentor -> 'preferred_mentee_exp_level'
> * MentorUpdate -> 'preferred_mentee_exp_level'  and 'profile_id'
> * Mentee -> ' experience_level'
> * MenteeUpdate ->  ' experience_level' and 'profile_id'

Disregard the previous comment, another PR was created to work in this changes. Excellent work!

**Review by GermanParra — APPROVED** [review]

**Comment by mikayla-kosmala** [comment]
> Let's remove the skill level field, as per our last talk with Jason.

Do we still want to remove them if we have a pull request working on this?
[#119](https://github.com/BloomTech-Labs/underdog-devs-ds-a/pull/119)

**Review by BrokenShell — APPROVED** [review]

**Pull request merged**